packages/neon/neon/lib/src/app.dart

```diff
@@ -267,9 +267,8 @@ class _NeonAppState extends State<NeonApp> with WidgetsBindingObserver, tray.Tra
 
                 FlutterNativeSplash.remove();
                 return ResultBuilder<Capabilities?>(
-                  stream: activeAccountSnapshot.hasData
-                      ? widget.accountsBloc.getCapabilitiesBloc(activeAccountSnapshot.data!).capabilities
-                      : null,
+                  stream:
+                      activeAccountSnapshot.hasData ? widget.accountsBloc.activeCapabilitiesBloc.capabilities : null,
```

Comment:
This is a bit odd because you are checking one variable but then accessing something differently.

Reply:
Yeah the difference between the comments on the interface and the implementation ...

But you're right

Reply:
How would that look like?